### PR TITLE
[CI] Sets up a workflow per project

### PR DESCRIPTION
**Pull request by chrisbanes**

### CODE REVIEW

File: .github/workflows/JetNews.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+name: JetNews
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'JetNews/**'
```
Comment (JoseAlcerreca) [inline .github/workflows/JetNews.yaml:8]: I assume you can't use SAMPLE_PATH here yet?

Comment (chrisbanes) [reply .github/workflows/JetNews.yaml:8]: Nope, `env` is a runtime thing and these are statically read.

Comment (chrisbanes) [reply .github/workflows/JetNews.yaml:8]: I'll add some comments once I've got it working

**Comment by JoseAlcerreca** [comment]
Runs JetNews only :+1: Now let's trigger for another project

**Comment by JoseAlcerreca** [comment]
Runs both projects correctly :+1:

**Review by JoseAlcerreca — APPROVED** [review]

**Comment by JoseAlcerreca** [comment]
image

**Pull request merged**